curator/src/mail_agent/newsletter_script.ts

```diff
@@ -0,0 +1,31 @@
+import { curateAndGenerateNewsletter } from './newsletter_txt_format';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+var postmark = require("postmark");
+
+// Calls the curateAndGenerateNewsletter function with right parameters then sends the mail
+// TODO : Add all the dynamic part (newsletter parameters, email params)
+
+async function runNewsletter() {
+    const {markdown, html} = await curateAndGenerateNewsletter();
+    //console.log(markdown);  // uncomment this part and comment below to test without postmark
+
+    // Sending email :
+    var client = new postmark.ServerClient(process.env.POSTMARK_SERVER_API_KEY as string);
+    var mail = process.env.POSTMARK_DEFAULT_MAIL as string;
+
+    client.sendEmail({
```

Comment:
You should lint this file. You can follow [this tutorial](https://www.digitalocean.com/community/tutorials/workflow-auto-eslinting) to enable lint on save on VSCode